modules/nw-ingress-default-internal.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+[id="nw-ingress-default-internal_{context}"]
+= Configuring the default Ingress Controller for your cluster to be internal
+
+You can configure the `default` Ingress Controller for your cluster to be internal by deleting and recreating it.
+
+[WARNING]
+====
+If your cloud provider is Azure, you must have at least one public load balancer that points to your nodes.
+If you do not, all of your nodes will lose egress connectivity to the Internet.
```

Comment:
Alex and I discussed our lack of a guideline for "internet" vs. "Internet". Our docs use both.  ISG has weird guidance about using it both ways, but it's pretty outdated. Elsewhere, I most often see it lowercase, which I'm a fan of. (Check out goog's explanation: https://developers.google.com/style/word-list#letter-i)

Reply:
Oh thank goodness. That's my preference too. :)